.gitignore

```diff
@@ -358,4 +358,5 @@ samples/**/*.cs
 samples/**/*.txt
 samples/**/*.csproj
 samples/**/*.sln
-test/**/*.yaml
\ No newline at end of file
+test/**/*.yaml
+test/TestServerProjects/**/Generated/Azure.Core.Shared/*.cs
```

Comment:
We can use Directory.Props to include shared file into project without having them copied around locally.

Reply:
You would also not need the exclude `Exclude="../TestServerProjects/**/Azure.Core.Shared/*.cs"`

Reply:
Done..